src/client/containers/CartPage/CartPage.Container.js

```diff
@@ -1,10 +1,94 @@
 import React from 'react';
 import './CartPage.Style.css';
+import { useParams } from 'react-router-dom';
+import Loader from '../../components/Loader/index';
+import ShoppingItem from '../../components/ShoppingItem/ShoppingItem';
+import DeliveryInfo from '../../components/DeliveryInfo/DeliveryInfo.component';
+import TotalPrice from '../../components/TotalPriceCard/TotalPriceCard.component';
+import ContactForm from '../../components/ContactForm/ContactForm.component';
+import ButtonComponent from '../../components/Button/Button.component';
+import { useFetchApi } from '../../hooks/UseFetchApi';
 
 const CartPageContainer = () => {
+  const [cartItem, setCartItem] = React.useState([]);
+  const [user, setUser] = React.useState({});
+  const [userId, setUserId] = React.useState('');
+  const [total, setTotal] = React.useState(0);
+  const { id } = useParams();
+
+  const orderData = useFetchApi(`orders/${id}`);
+
+  React.useEffect(() => {
+    if (!orderData.isLoading) {
+      setCartItem(orderData.data.items);
+      setUserId(orderData.data.order.userId);
+    }
+  }, [orderData]);
+
+  const userInfo = useFetchApi(`users/${userId}`);
+
+  React.useEffect(() => {
+    if (!userInfo.isLoading) {
+      setUser(userInfo.data[0]);
+    }
+  }, [userInfo, user]);
+
+  React.useEffect(() => {
+    let cost = 0;
+    const subTotal = cartItem.map((item) => item.price * item.quantity);
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < subTotal.length; i++) {
+      cost += subTotal[i];
+    }
+    setTotal(cost);
+  }, [cartItem]);
+
   return (
     <div>
-      <h1>Cart page container</h1>
+      <h1>SHOPPING CART</h1>
+      <div className="main">
+        <div className="left">
+          <div className="item-list">
+            {orderData.isLoading ? (
+              <Loader />
+            ) : (
+              cartItem.map((item) => {
+                return (
+                  <ShoppingItem
+                    productName={item.name}
+                    quantity={item.stock_amount}
+                    price={item.price}
+                    productImg={item.picture}
+                  />
+                );
+              })
+            )}
+          </div>
+          <div className="delivery-info">
+            {userInfo.isLoading ? (
+              <Loader />
+            ) : (
+              <DeliveryInfo editMode={true} vertDisplay={false} user={user} />
+            )}
+          </div>
+        </div>
+        <div className="right">
+          <div className="total-price">
+            <TotalPrice subTotal={total} />
+          </div>
+          <div className="contact">
+            <ContactForm fullName="Varsha" email="varsha.verma@gmail.com" />
```

Comment:
I think you should get name and email information from firebase context.

Reply:
Firebase still have some error.. so did not changes this stuff yet
once firebase error are fixed, will change accordingly